manuscript/chapter1.md

```diff
@@ -1,44 +1,41 @@
 # Introduction to React
 
-The chapter gives you an introduction to React. You may ask yourself: Why should I learn React in the first place? This chapter seeks to answer that question. Afterward, you will dive into the ecosystem by bootstrapping your first React application from scratch with zero-configuration. Along the way, you will get an introduction to JSX and ReactDOM. So be prepared for your first React components.
+This chapter is an introduction to React, a JavaScript library for rendering interfaces in single-page and mobile applications. I explain why developers should consider adding the React library to their toolbelts. We will dive into the React ecosystem, creating your first React application from scratch with no configuration. Along the way, we will introduce JavaScript XML(**JSX**) and **ReactDOM**, so you have an understanding of React's practical uses before the next chapter.
```

Comment:
maybe: ... we will introduce **JSX**, the syntax for React, and Rect **DOM**, so you have ...